ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/LedgerApiTestTool.scala

```diff
@@ -191,7 +191,15 @@ object LedgerApiTestTool {
         new ColorizedPrintStreamReporter(
           System.out,
           config.verbose,
-        ).report(summaries, excludedTestSummaries, identifierSuffix)
+        ).report(
+          summaries,
+          excludedTestSummaries,
+          Seq(
+            "identifierSuffix" -> identifierSuffix,
+            "concurrentTestRuns" -> config.concurrentTestRuns.toString,
+            "timeoutScaleFactor" -> config.timeoutScaleFactor.toString,
```

Comment:
Any reason why these need to be camelCase?

```suggestion
            "identifier suffix" -> identifierSuffix,
            "concurrent test runs" -> config.concurrentTestRuns.toString,
            "timeout scale factor" -> config.timeoutScaleFactor.toString,
```

Reply:
No, other than "that's how it was before".

Reply:
I'll leave it as is, the labels correspond 1:1 to the config fields, which makes them easier to search for.